packages/botonic-react/src/components/multichannel/multichannel-button.jsx

```diff
@@ -1,6 +1,7 @@
 import React, { useContext } from 'react'
 
 import { RequestContext } from '../../contexts'
+import { truncateText } from '../../util/strings'
```

Comment:
this one can also be imported just from `'../../util'` instead of including string right?

Reply:
You're right! Changed!